extensions/logging-sentry/runtime/src/main/java/io/quarkus/sentry/SentryConfig.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.sentry;
+
+import java.util.Optional;
+import java.util.logging.Level;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(phase = ConfigPhase.RUN_TIME)
+public class SentryConfig {
+
+    /**
+     * Sentry DSN
+     *
+     * The DSN is the first and most important thing to configure because it tells the SDK where to send events. You can find
+     * your project’s DSN in the “Client Keys” section of your “Project Settings” in Sentry.
+     */
+    @ConfigItem
+    public String dsn;
+
+    /**
+     * The sentry log level.
+     */
+    @ConfigItem(defaultValue = "WARNING")
+    public Level level;
+
+    /**
+     * Sentry differentiates stack frames that are directly related to your application (“in application”) from stack frames
+     * that come from other packages such as the standard library, frameworks, or other dependencies. The difference is visible
+     * in the Sentry web interface where only the “in application” frames are displayed by default.
+     *
+     * You can configure which package prefixes your application uses with this option, which takes a comma separated list.
+     *
+     * This option is highly recommended as it affects stacktrace grouping and display on Sentry. See documentation:
+     * https://docs.sentry.io/clients/java/config/#in-application-stack-frames
+     */
+    @ConfigItem
+    public Optional<String> packages;
```

Comment:
Shouldn't this be a `Optional<List<String>>`?

Reply:
@dmlloyd Sentry then needs a comma separated list. Using a `Optional<List<String>>` would mean converting to be deconverted just after, still a good idea?

Reply:
I think it's important for all of our lists to be formatted the same way whenever possible.

Reply:
Ok, done. I like consistency too ;-)